_episodes/07-thresholding.md

```diff
@@ -14,532 +14,370 @@ threshold an image represented by a numpy array."
+- "Thresholding produces a binary image, where all pixels with intensities
 above (or below) a threshold value are turned on, while all other pixels are
 turned off."
 - "The binary images produced by thresholding are held in two-dimensional NumPy
-arrays, since they have only one color value channel. They are boolean, hence they contain 
+arrays, since they have only one color value channel. They are boolean, hence they contain
 the values 0 (off) and 1 (on)."
-- "Thresholding can be used to create masks that select only the interesting 
-parts of an image, or as the first step before 
-[Edge Detection]({{ page.root }}/08-edge-detection/) or finding 
+- "Thresholding can be used to create masks that select only the interesting
+parts of an image, or as the first step before
+[Edge Detection]({{ page.root }}/08-edge-detection/) or finding
 [Contours]({{ page.root }}/09-contours/)."
 ---
 
-In this episode, we will learn how to use skimage functions to apply 
+In this episode, we will learn how to use skimage functions to apply
 thresholding to an image. Thresholding is a type of *image segmentation*,
-where we change the pixels of an image to make the image easier to 
-analyze. In thresholding, we convert an image from color or grayscale into a 
-*binary image*, i.e., one that is simply black and white. Most frequently, we 
-use thresholding as a way to select areas of interest of an image, while 
-ignoring the parts we are not concerned with. We have already done some simple 
-thresholding, in the "Manipulating pixels" section of the 
+where we change the pixels of an image to make the image easier to
+analyze. In thresholding, we convert an image from color or grayscale into a
+*binary image*, i.e., one that is simply black and white. Most frequently, we
+use thresholding as a way to select areas of interest of an image, while
+ignoring the parts we are not concerned with. We have already done some simple
+thresholding, in the "Manipulating pixels" section of the
 [Skimage Images]({{ page.root }}/03-skimage-images/) episode. In that case, we
 used a simple NumPy array manipulation to separate the pixels belonging to the
-root system of a plant from the black background. In this episode, we will 
+root system of a plant from the black background. In this episode, we will
 learn how to use skimage functions to perform thresholding. Then, we will use the
-masks returned by these functions to select the parts of an image we are 
-interested in. 
+masks returned by these functions to select the parts of an image we are
+interested in.
 
 ## Simple thresholding
 
-Consider this image, with a series of crudely cut shapes set against a white 
-background. The black outline around the image is not part of the image.
+Consider the image `fig/06-junk-before.jpg` with a series of
+crudely cut shapes set against a white background. The black outline
+around the image is not part of the image.
```

Comment:
> The black outline
> around the image is not part of the image.

If we merge in the latest version of the lesson template, we can remove the border and use the `.image-with-shadow` class for this image https://carpentries.github.io/lesson-example/04-formatting/index.html#applying-a-shadow-to-images

Reply:
Great suggestion. I will see if I can manage to merge the updated lesson template.

Reply:
Best kept to a separate PR, I think, lest the diff on this one gets too large.

Reply:
I have opened #158 to track this separately.